api/appointments.ts

```diff
@@ -0,0 +1,10 @@
+const handler = require('./handler.ts');
```

Comment:
This needs to be switched to the new style as in https://github.com/movshov/VA-Audiology-App/pull/10